.github/renovate.json

```diff
@@ -12,6 +12,14 @@
     "org.apache.maven.resolver:maven-resolver-transport-file",
     "org.apache.maven.resolver:maven-resolver-transport-http"
   ],
+  "packageRules": [
+    {
+      "allowedVersions": "<7.0.0",
+      "matchPackageNames": ["com.google.inject:guice"],
+      "description": "We focus on Guice 6 until core adapts 7",
+      "enabled": false
```

Comment:
Why is this rule disabled? I could understand disabling it if we did not want to get PRs for this package, but in fact we do want to get PRs for version 6.x of this package.